types.d.ts

```diff
@@ -4050,7 +4050,7 @@ declare interface Environment {
 declare class EnvironmentPlugin {
 	constructor(...keys: (string | string[] | Record<string, string>)[]);
```

Comment:
@102: Wouldn't it also make sense to update the constructor parameters to also allow type `any` values. This would fix issue #18719